metastore/src/java/org/apache/hadoop/hive/metastore/ReplChangeManager.java

```diff
@@ -250,28 +247,27 @@ static Path getCMPath(Configuration conf, String checkSum)
    * Get original file specified by src and chksumString. If the file exists and checksum
    * matches, return the file; otherwise, use chksumString to retrieve it from cmroot
    * @param src Original file location
-   * @param chksumString Checksum of the original file
-   * @param conf
+   * @param checksumString Checksum of the original file
+   * @param hiveConf
    * @return Corresponding FileStatus object
-   * @throws MetaException
    */
-  static public FileStatus getFileStatus(Path src, String chksumString,
```

Comment:
the idea is to give most restrictive scope visibility for methods / varaibles/ classes. Depending on what where its accessed for now the visibility has been changed